modules/core/arrow-test/src/main/kotlin/arrow/test/laws/MonadLaws.kt

```diff
@@ -95,25 +95,25 @@ object MonadLaws {
 
   fun <F> Monad<F>.monadComprehensions(EQ: Eq<Kind<F, Int>>): Unit =
     forAll(Gen.int()) { num: Int ->
-      binding {
-        val a = just(num).bind()
-        val b = just(a + 1).bind()
-        val c = just(b + 1).bind()
+      fx {
+        val (a) = just(num)
+        val (b) = just(a + 1)
+        val (c) = just(b + 1)
```

Comment:
is this syntax an option to bind or the new only one? and how that's works?

Reply:
It's an option to bind. Works by overriding `component1` for all `Kind<F, A>` resulting in a suspended `A`